src/pages/Register/Components/BgPic.tsx

```diff
@@ -0,0 +1,56 @@
+import { Box, Grid, Image, Text } from "@chakra-ui/react";
+import React from "react";
+
+const BgPics = () => {
+  return (
+    <Grid
+      templateRows="repeat(2, 1fr)"
+      templateColumns="repeat(2, 1fr)"
+      maxWidth="400"
+      maxH="700"
+    >
+      <Box padding="1">
+        <Image src="https://d3ptyyxy2at9ui.cloudfront.net/assets/images/ae01c415678c90d0c606bc6917758502.jpg" />
+        <Box mt="5" ml="10">
+          <Text as="b" fontSize="sm">
+            30 million+{" "}
+          </Text>
+          <Text fontSize="sm">app dowloads </Text>
+        </Box>
+      </Box>
+
+      <Box padding="1">
+        <Image src="https://d3ptyyxy2at9ui.cloudfront.net/assets/images/573fa908fbccdf1600f90f16a0c2e311.jpg" />
+        <Box margin="5" ml="10">
+          <Text as="b" fontSize="sm">
+            {" "}
+            15 years+{" "}
+          </Text>
+          <Text fontSize="sm"> in business </Text>
+        </Box>
+      </Box>
+
+      <Box padding="1">
+        <Image src="https://d3ptyyxy2at9ui.cloudfront.net/assets/images/87ded242639ba32ef4d0e6c21c9f30aa.jpg" />
+        <Box margin="5" ml="10">
+          <Text as="b" fontSize="sm">
+            2 billion+
+          </Text>
+          <Text fontSize="sm">tasks completed</Text>
+        </Box>
+      </Box>
+
+      <Box padding="1">
+        <Image src="https://d3ptyyxy2at9ui.cloudfront.net/assets/images/ba90575091d9eafd60f851f7297cb202.jpg" />
+        <Box margin="5" ml="10">
+          <Text as="b" fontSize="sm">
+            1 million+{" "}
+          </Text>
+          <Text fontSize="sm">pro users </Text>
+        </Box>
+      </Box>
+    </Grid>
+  );
+};
+
+export default BgPics;
```

Comment:
As you can see in this file, you are repeating yourself.

   ```
<Box padding="1">
        <Image src="https://d3ptyyxy2at9ui.cloudfront.net/assets/images/ae01c415678c90d0c606bc6917758502.jpg" />
        <Box mt="5" ml="10">
          <Text as="b" fontSize="sm">
            30 million+{" "}
          </Text>
          <Text fontSize="sm">app dowloads </Text>
        </Box>
      </Box>
```

Create a generic component and pass the info as a prop.

```
   <Box padding="1">
        <Image src={props} />
        <Box mt="5" ml="10">
          <Text as="b" fontSize="sm">
       {props}
          </Text>
          <Text fontSize="sm">{props}</Text>
        </Box>
      </Box>
```

Reply:
i didnt understand what am i supposed to do or not to do?

Reply:
i think i got it . with ur props video's help i did used props and no more repeats.